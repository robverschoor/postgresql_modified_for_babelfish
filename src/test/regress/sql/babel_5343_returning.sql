CREATE TABLE toverflow(id SERIAL, col1 VARCHAR);
INSERT INTO toverflow VALUES (default, 'hope') RETURNING xmax::text::int;
INSERT INTO toverflow VALUES (default, 'jack') RETURNING xmax;
SELECT * FROM toverflow ORDER BY id;
DROP TABLE toverflow;
