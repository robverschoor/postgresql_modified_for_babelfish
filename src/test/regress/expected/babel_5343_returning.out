CREATE TABLE toverflow(id SERIAL, col1 VARCHAR);
INSERT INTO toverflow VALUES (default, 'hope') RETURNING xmax::text::int;
 xmax 
------
    0
(1 row)

INSERT INTO toverflow VALUES (default, 'jack') RETURNING xmax;
 xmax 
------
    0
(1 row)

SELECT * FROM toverflow ORDER BY id;
 id | col1 
----+------
  1 | hope
  2 | jack
(2 rows)

DROP TABLE toverflow;
